python/examples/DFtoVW_tutorial.ipynb

```diff
@@ -0,0 +1,1521 @@
+    "\n",
+    "The current section illustrates the basic usage. Section 4 will present the advanced usage.\n",
+    "\n",
+    "To use the `DFtoVW.from_colnames` method: the user provides the target name as a string (parameter `y`), the features as a list of string (`x`) and the dataframe (`df`) that has to be converted to the method. He can also specify the type of label (argument `label_type`, default being `simple_label` for the regression). This call will build a `DFtoVW` object. \n",
+    "\n",
+    "To get the actual examples (as a list), the user call the `convert_df` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "label: PM\n",
+      "features: ['DEWP', 'HUMI', 'Iws', 'PRES', 'TEMP', 'cbwd', 'city', 'day', 'hour', 'month', 'precipitation', 'season', 'year']\n"
+     ]
+    }
+   ],
+   "source": [
+    "y = \"PM\"\n",
+    "x = df_city.columns.difference([\"PM\", \"log_PM\"]).tolist()\n",
+    "\n",
+    "print(\"label:\", y)\n",
+    "print(\"features:\", x)\n",
+    "\n",
+    "converter = DFtoVW.from_colnames(df=df_city, y=y, x=x)\n",
+    "examples = converter.convert_df()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can inspect the first lines:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['106.0 | DEWP:-0.009151075492972885 HUMI:0.1860656151915359 Iws:-0.33742871431739124 PRES:0.03281683774820812 TEMP:-0.36894735027639736 cbwd=NW city=beijing day=17 hour=11 month=4 precipitation:-0.0025309449430064956 season=1.0 year=2014',\n",
+       " '50.0 | DEWP:0.15081700519152547 HUMI:0.35149670347508516 Iws:-0.4527764771757221 PRES:-0.7503776051439223 TEMP:0.2438891818683882 cbwd=SE city=guangzhou day=29 hour=18 month=11 precipitation:-0.0025309449430064956 season=3.0 year=2012',\n",
+       " '28.0 | DEWP:0.24758781943276517 HUMI:0.07191816427588703 Iws:-0.3217955010362255 PRES:-0.6611529217764663 TEMP:1.0030747366149138 cbwd=SW city=chengdu day=26 hour=20 month=6 precipitation:-0.0025309449430064956 season=2.0 year=2014']"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "examples[:3]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For categorical features, we see that VW format is `feature_name=feature_value` whereas for numerical feature the format is `feature_name:feature_value`. \n",
+    "\n",
+    "Also note that:\n",
+    "- for categorical variable, VW add `:1` behind the scene. For instance `day=14` is equivalent to `day=14:1`\n",
```

Comment:
nit: 
```suggestion
    "- for categorical variables, VW adds `:1` behind the scene. For instance `day=14` is equivalent to `day=14:1`\n",
```

Reply:
Thank you for carefully rereading haha :)

 fixed in commit 6115e12